views/partials/layout.njk

```diff
@@ -5,7 +5,12 @@
 {% block head %}
 
 <!--[if !IE 8]><!-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```

Comment:
why grab some content from googleapis and others direct from jquery website?

Reply:
I'll grab them all from the jquery site and introduce SRI checking.